src/background.js

```diff
@@ -54,10 +58,31 @@ function sendMetrics(event, context = {}) {
       };
     }
 
+    // Generate cd10 based on footer.js rules
+    if (sync.email) { // If user is authenticated
+      if (sync.error) {
+        metrics.cd10 = 'error';
+      } else if (sync.isSyncing) {
+        metrics.cd10 = 'isSyncing';
+      } else {
+        metrics.cd10 = 'synced';
+      }
+    } else {
+      if (sync.isOpeningLogin) { // eslint-disable-line no-lonely-if
+        metrics.cd10 = 'openLogin';
+      } else if (sync.isPleaseLogin) {
+        metrics.cd10 = 'verifyAccount';
+      } else if (sync.isReconnectSync) {
+        metrics.cd10 = 'reconnectSync';
+      } else {
+        metrics.cd10 = 'signIn';
+      }
+    }
+
     return analytics.sendEvent('notes', event, metrics);
   };
   clearTimeout(timeouts[event]);
-  timeouts[event] = setTimeout(later, 20000);
+  timeouts[event] = setTimeout(later, 4000);
```

Comment:
Oups, I did change that value for testing purpose but should stay at 20000. Will fix in a second.